### DISCUSSION TITLE
@next/font to support custom fonts hosted in web

### DISCUSSION THREAD

**Discussion by karlomedallo** [vercel/next.js#43327] [Ideas]
### Describe the feature you'd like to request

Similar to localFont, I would like to request for fonts hosted in a remote server (e.g. CDN) to be loaded. 

Thanks!

### Describe the solution you'd like

Whether that would be something like remoteFont() or just allow src in localFont to accept URLs from a remote server.

Use case here is that CMS administrators can create themes in CMS with custom fonts. When the font is uploaded in a CMS, they can be accessed remotely via a URL.

### Describe alternatives you've considered

I understand that sometimes, these CMS/Media Managers would need to have some auth/custom headers to be passed across. That can be an improvement to this but initially not required

**Comment by jarPotato** [comment]
Any updates on this?
